docker/Dockerfile

```diff
@@ -1,5 +1,6 @@
 FROM ubuntu:22.04
 
+ARG DEVCONTAINER
```

Comment:
What does this do? Is this something specific for the devcontainer to work?

Reply:
This is an old artifact that is no longer used. I will remove it